boxoffice/views/admin.py

```diff
@@ -35,3 +38,16 @@ def jsonify_org(data):
 @render_with({'text/html': 'index.html', 'application/json': jsonify_org}, json=True)
 def org(organization):
     return dict(org=organization, title=organization.title)
+
+
+@app.route('/admin/o/<org>/revenue')
+@load_models(
+    (Organization, {'name': 'org'}, 'organization')
+    )
+def org_revenue(organization):
+    if not request.args.get('access_token') or request.args.get('access_token') != organization.details.get("access_token"):
+        abort(401)
+    item_collection_ids = [item_collection.id for item_collection in organization.item_collections]
+    timezone = g.user.timezone if g.user else app.config.get('TIMEZONE')
```

Comment:
I'd say you have a problem here that needs resolution. The timezone you're returning data for needs to be consistent. It should be a call parameter.